console/module/user/src/main/java/org/eclipse/kapua/app/console/module/user/shared/util/GwtKapuaUserModelConverter.java

```diff
@@ -25,11 +25,13 @@
 import org.eclipse.kapua.model.KapuaEntity;
 import org.eclipse.kapua.model.query.predicate.AttributePredicate.Operator;
 import org.eclipse.kapua.service.user.UserFactory;
+import org.eclipse.kapua.service.user.UserPredicates;
 import org.eclipse.kapua.service.user.UserQuery;
 import org.eclipse.kapua.service.user.UserStatus;
 
-import org.eclipse.kapua.app.console.module.api.shared.util.GwtKapuaCommonsModelConverter;
-import org.eclipse.kapua.service.user.UserPredicates;
+import com.extjs.gxt.ui.client.Style.SortDir;
```

Comment:
Please do not change position of imports. SortDir for example was already there as first import.